wptagent.py

```diff
@@ -901,6 +915,11 @@ def main():
             print "No browsers configured. Check that browsers.ini is present and correct."
             exit(1)
 
+    if options.collectversion and \
+            platform.system() == "Windows":
+        versions = get_browser_versions(browsers)
+        options.browser_versions = versions
```

Comment:
It feels a little strange to be appending the version info to the options. I won't hold it back for just this but adding a 'version' entry to each entry in browsers, passing the browsers into the webpagetest instance and building the string in there feels cleaner if it's not too painful.

Reply:
That's actually how I had originally done it, but I wanted to minimize the changes if the option was not enabled.  But it makes more logical sense to use the browser object for browser attributes and just pass it in.